modules/operation-measure-evaluate-measure/deps.edn

```diff
@@ -34,13 +34,33 @@
    {blaze/db-stub
     {:local/root "../db-stub"}}}
 
+  ;; For both toying at the terminal and editor support (via nREPL)
+  ;; Supports Emacs' refactor-nrepl and sayid
+  :repl
+  {:extra-deps {nrepl/nrepl                   {:mvn/version "1.3.0"}
+                cider/cider-nrepl             {:mvn/version "0.49.3"}
+                refactor-nrepl/refactor-nrepl {:mvn/version "3.10.0"}
+                com.billpiel/sayid            {:mvn/version "0.1.0"}}
+   :main-opts  ["--main" "nrepl.cmdline"
+                "--middleware" "[cider.nrepl/cider-middleware, com.billpiel.sayid.nrepl-middleware/wrap-sayid, refactor-nrepl.middleware/wrap-refactor]"
+                "--interactive"]}
+
   :kaocha
   {:extra-deps
    {lambdaisland/kaocha
     {:mvn/version "1.91.1392"}}
 
    :main-opts ["-m" "kaocha.runner"]}
 
+  :kaocha-watch
+  {:extra-deps
+   {lambdaisland/kaocha
+    {:mvn/version "1.91.1392"}}
+   :exec-fn   kaocha.runner/exec-fn
```

Comment:
I don't know about this new kind of execution config. But if that is now preferred over `:main-opts` we may use it everywhere?

Reply:
I certainly agree in your consistency argument. That being said, I am not sure whether there is a preferred way at all or not - much less which one that would be.

So, I will leave this one open for now so I can research it further later on.

Reply:
For now, I will leave this change outside of this PR.
I will review this later.

Reply:
See https://github.com/samply/blaze/issues/2105